packages/theme-default/src/components/Card/index.tsx

```diff
@@ -0,0 +1,36 @@
+interface CardProps {
```

Comment:
We should add props related in both zh and en docs to tell users what they can configure.

![image](https://github.com/user-attachments/assets/43ea209a-0e8d-4b7f-8fef-c132888e8f0c)